src/lib/ext-monitor.js

```diff
@@ -1,6 +1,10 @@
 import { getActivityLogPageURL } from './ext-listen.js';
 
 export default class ExtensionMonitor {
+  // Map<string, Array>
+  loadedLogs = new Map();
+  // Map<number, string>
+  loadedLogsTabIds = new Map();
```

Comment:
Given that we agreed to clear the logs automatically when the tab we have opened has been navigated or closed, it may be better to:

- keep the loaded logs in a map keyed by tabId, as the value of the map we could store an object that contains both the filename and the loaded logs in two separate properties

does that sounds reasonable?

(this should, at least at a first glance, also simplify a bit `setLoadedLogs`, e.g. we will not need to care about the unique file name thing)

Reply:
Yeah, sounds good :-)